test/unit/presenters/publishing_api/payload_builder/first_public_at_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+module PublishingApi
+  module PayloadBuilder
+    class PayloadBuilderFirstPublicAtTest < ActiveSupport::TestCase
+      def test_returns_a_draft_publications_parent_document_created_at
+        draft_publication = create(:draft_publication) do |pub|
```

Comment:
By using a publication fixture we're implying a coupling that doesn't exist.

With duck typing `FirstPublicAt` only needs to know about an object that responds to `first_published_at` or responds to `document` with an object that responds to `created_at`.

Maybe something like:

```ruby
def test_returns_first_published_at_if_present
  first_published_at = Object.new
  item = stub(first_published_at: first_published_at)

  assert_equal { first_public_at: first_published_at }, FirstPublicAt.for(item)
end

def test_returns_document_created_at_for_nil_first_published_at
  created_at = Object.new
  item = stub(first_published_at: nil, document: stub(created_at: created_at))

  assert_equal { first_public_at: created_at }, FirstPublicAt.for(item)
end
```